curriculum/js1/7.js

```diff
@@ -6,9 +6,31 @@
  */
 
 const solution = (a) => {
-  return 0;
+  if(a === 1) return false;
```

Comment:
this logic is really confusing, could you use the `%` for all numbers from 2 to a?